src/pocketmine/plugin/PluginBase.php

```diff
@@ -229,7 +229,7 @@ public function getResources() : array{
 		$resources = [];
 		if(is_dir($this->file . "resources/")){
 			foreach(new \RecursiveIteratorIterator(new \RecursiveDirectoryIterator($this->file . "resources/")) as $resource){
-				$resources[] = $resource;
+				$resources[substr($resource, strlen($this->file . "resources/"))] = $resource;
```

Comment:
this probably won't work as you expect on Windows (different directory separator), please check it.